CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_hubspot_source v0.16.0
+[PR #129](https://github.com/fivetran/dbt_hubspot_source/pull/129) includes the following updates:
+
+## Breaking Changes
+- Switched from using the `fivetran_utils.remove_prefix_from_columns` macro to the `hubspot_source.remove_duplicate_and_prefix_from_columns` macro for when `hubspot__pass_through_all_columns` is enabled and you are passing through all columns in the `stg_hubspot__company`, `stg_hubspot__contact`, `stg_hubspot__deal`, and `stg_hubspot__ticket` models. This also ensures the source fields passed through are all quoted from the onset. This is a breaking change because this macro can remove duplicate fields, resulting in an impact to your schema. See the [v0.12.0 release notes](https://github.com/fivetran/dbt_hubspot_source/releases/tag/v0.12.0) for more information.
```

Comment:
I don't think we necessarily need to link the v0.12.0 release notes since that may cause more confusion. I think what you have before is great, no need to add the last sentence.

Reply:
Also the `stg_hubspot__ticket` model was not updated in this release for this change. You can include this in the below bugfix however.